app/controllers/productions_controller.rb

```diff
@@ -0,0 +1,66 @@
+class ProductionsController < ApplicationController
+  before_action :set_production, only: [:show, :update, :destroy]
+
+  def index
+    p params
+    authenticate_user!
+    set_productions
+    render :index
+  end
+
+  def create
+    p "*" * 40
+    p params.inspect
+    set_productions
+    @production = Production.new(name: params[:production][:name], date: params[:production][:date], designer_id: current_user.id)
+    if @production.save
+      render :index
+    else
+      redirect_to :root
```

Comment:
put error messages in the `flash` before redirecting and display the errors to the user